src/js/chat/templates.js

```diff
@@ -42,7 +42,7 @@ var linkify = exports.linkify = function(message) {
 };
 
 var escapeEmoteCode = function(code) {
-    return code.replace(/('|"|>|<|&)/g, '');
```

Comment:
This opens us up to potential xss in the event someone doesn't handle the data properly.. If you want to save the character it has to be encoded.